config/katello.migrations/200803163936-add-statistics-plugin.rb

```diff
@@ -0,0 +1 @@
+answers['foreman::plugin::statistics'] ||= false
```

Comment:
Not sure how much it matters, but you're missing a `20` in the file name

Reply:
It was added by the script, in katello they all skip the leading `20`.

Reply:
ok, then we should be good until 2100 ;)